src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -351,15 +351,14 @@ export default class OrgPerson extends Mixins(EntityFilterMixin, CommonMixin) {
 
   /** Whether the Incorporator role should be disabled. */
   private get disableIncorporatorRole (): boolean {
-    // disable this role if it's locked (ie, pre-selected for a new person)
-    // or if this is an org, which can only have this role
-    return (this.isRoleLocked(RoleTypes.INCORPORATOR) || this.isOrg)
```

Comment:
This worked for adding and editing an incorporator/org...

But the check for Director, below, could not be done the same way. With a fresher mind, I simply simplified the logic, which handles all cases.